content/pages/office-policy.md

```diff
@@ -0,0 +1,42 @@
+Title: Office Policy
+Category: pages
+Status: hidden
+Slug: office-policy
+
+(Version 2.0 March 8th 2015)
+
+# Personnel #
+
+* If the office is open for office hours, drop-ins or for any other reason, a MINIMUM of ONE committee member of WiCS must be present in the office at all times.
```

Comment:
This should be 80-char justified.

I'm also not a fan of the use of caps in this document. I'd recommend using a bold font instead.